esp/logging/loggingmanager/loggingmanager.hpp

```diff
@@ -84,6 +84,7 @@ class CLoggingManager : implements ILoggingManager, public CInterface
     CLogContentFilter logContentFilter;
     unsigned serviceMask = 0;
 
+    void updateServiceMask(LOGServiceType service, bool value = true);
```

Comment:
There's a recent discussion about avoiding using default params, can you remove it?